spec/controllers/admin/customers_controller_spec.rb

```diff
@@ -66,6 +66,11 @@ def default_sorting_direction
         expect(locals[:resources].map(&:name)).to eq sorted_customer_names
       end
     end
+
+    it "assigns the action name as a string" do
+      get :index
+      expect(controller.action_name).to be_a(String)
+    end
```

Comment:
Can you also remove this, please? This is standard Rails behaviour and unrelated to this feature, so it doesn't need to be tested. Sorry I missed this yesterday.

Reply:
I want make sure action name is string not symbo, anyway I can remove it
```
      action =
        case action
        when "update" then "edit"
        when "create" then "new"
        else action
        end
```